packages/layers/src/utils.js

```diff
@@ -146,3 +146,113 @@ export function makeBoundingBox(viewState) {
     viewport.unproject([0, viewport.height])
   ];
 }
+
+const TARGETS = [1, 2, 3, 4, 5, 10, 20, 25, 50, 100, 200, 250, 500, 1000];
+const MIN_TARGET = TARGETS[0];
+const MAX_TARGET = TARGETS[TARGETS.length - 1];
+
+const SI_PREFIXES = [
+  { symbol: 'Y', exponent: 24 },
+  { symbol: 'Z', exponent: 21 },
+  { symbol: 'E', exponent: 18 },
+  { symbol: 'P', exponent: 15 },
+  { symbol: 'T', exponent: 12 },
+  { symbol: 'G', exponent: 9 },
+  { symbol: 'M', exponent: 6 },
+  { symbol: 'k', exponent: 3 },
+  { symbol: 'h', exponent: 2 },
+  { symbol: 'da', exponent: 1 },
+  { symbol: '', exponent: 0 },
+  { symbol: 'd', exponent: -1 },
+  { symbol: 'c', exponent: -2 },
+  { symbol: 'm', exponent: -3 },
+  { symbol: 'µ', exponent: -6 },
+  { symbol: 'n', exponent: -9 },
+  { symbol: 'p', exponent: -12 },
+  { symbol: 'f', exponent: -15 },
+  { symbol: 'a', exponent: -18 },
+  { symbol: 'z', exponent: -21 },
+  { symbol: 'y', exponent: -24 }
+];
+
+/**
+ * Convert a size value to meters.
+ * @param {number} size Size in original units.
+ * @param {string} unit String like 'mm', 'cm', 'dam', 'm', 'km', etc.
+ * @returns {number} Size in meters.
+ */
+export function sizeToMeters(size, unit) {
+  if (!unit || unit === 'm') {
+    // Already in meters.
+    return size;
+  }
+  if (unit.length > 1) {
+    let unitPrefix = unit.substring(0, unit.length - 1);
+    // Support 'u' as a prefix for micrometers.
+    if (unitPrefix === 'u') {
+      unitPrefix = 'µ';
+    }
+    // We remove the trailing 'm' from the unit, so 'cm' becomes 'c' and 'dam' becomes 'da'.
```

Comment:
Should this comment be on line 190? I'm not sure I see where the trailing 'm' is removed.